dbms/src/Storages/MergeTree/ReplicatedMergeTreeQueue.h

```diff
@@ -163,7 +164,7 @@ class ReplicatedMergeTreeQueue
       * If next_update_event != nullptr, will call this event when new entries appear in the log.
       * Returns true if new entries have been.
       */
-    bool pullLogsToQueue(zkutil::ZooKeeperPtr zookeeper, zkutil::EventPtr next_update_event);
+    bool pullLogsToQueue(zkutil::ZooKeeperPtr zookeeper, BackgroundSchedulePool::TaskHandle next_update_event);
```

Comment:
Should be next_update_task.

Reply:
Fixed in 31874ed17299c3d7b1c212d8077c147029ffe1fa

Reply:
What about merge_selecting_handle?

Reply:
Fixed in 50992843c6f51628c028a7b7bf4c2c1f346e4618

Reply:
Oops, you misunderstood me :) `next_update_task_handle` is perfectly OK, I was talking about the StorageReplicatedMergeTree member variable which is inconsistently named.

I won't comment several times if if there is a common issue with several separate places in code.

Reply:
I reverted prev. commit. You mean to replace merge_selecting_handle with merge_selecting_task_handle ?

Reply:
Yes